metadata/src/main/java/com/redhat/lightblue/metadata/parser/MetadataParser.java

```diff
@@ -181,6 +182,16 @@ public MetadataParser(Extensions<T> ex, TypeResolver typeResolver) {
         return this.extensions;
     }
 
+    private static Set<String> asSet(String... x) {
+        HashSet<String> set=new HashSet<>();
+        for(String a:x)
+            set.add(a);
```

Comment:
Done.

On Thu, Mar 31, 2016 at 5:33 PM, Dennis Crissman <notifications@github.com>
wrote:

> In
> metadata/src/main/java/com/redhat/lightblue/metadata/parser/MetadataParser.java
> <https://github.com/lightblue-platform/lightblue-core/pull/624#discussion_r58144635>
> :
>
> > @@ -181,6 +182,16 @@ public MetadataParser(Extensions<T> ex, TypeResolver typeResolver) {
> >          return this.extensions;
> >      }
> >
> > +    private static Set<String> asSet(String... x) {
> > +        HashSet<String> set=new HashSet<>();
> > +        for(String a:x)
> > +            set.add(a);
>
> Why not just: new HashSet(Arrays.asList(someArray));
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-core/pull/624/files/ea3a82c505fb57a14479315fd5b5ae19d4fdd0db#r58144635>
>